actix-rt/src/system.rs

```diff
@@ -57,10 +57,58 @@ impl System {
         Self::builder().name(name).build()
     }
 
-    #[allow(clippy::new_ret_no_self)]
-    /// Create new system using provided tokio Handle.
+    /// Create new system using provided tokio `LocalSet`.
     ///
     /// This method panics if it can not spawn system arbiter
+    ///
+    /// Note: This method uses provided `LocalSet` to create a `System` future only.
+    /// All the [`Arbiter`]s will be started in separate threads using their own tokio `Runtime`s.
+    /// It means that using this method currently it is impossible to make `actix-rt` work in the `Runtime` other
+    /// than the provided by `tokio` 0.2 (e.g. provided by `tokio_compat`).
```

Comment:
Reword "other than the provided".

IMO, it's also note clear what in `tokio_compat` you're referring to.

Reply:
Is it better now?

Reply:
yep, reads much better, thanks :)